### PR TITLE
fixd OldSample TwitterSearch

### PR DESCRIPTION
**Pull request by JohnHe404**
fixd TwitterSearch.Core in TwitterSearch

### CODE REVIEW

**Comment by martijn00** [comment]
Can you remove the code that is not used anymore?

**Comment by HeMinzhang** [comment]
@martijn00  OK! ~~

**Comment by HeMinzhang** [comment]
@martijn00  Can I keep **MvxTrace.Trace** ? Change to new log api in the future.

**Comment by martijn00** [comment]
@HeMinzhang MvxTrace should be removed. It is easy to replace with MvxLog

File: OldSamples/TwitterSearch/TwitterSearch.UI.Console/Properties/AssemblyInfo.cs [hunk 1]
```diff
@@ -1,35 +1,36 @@
 ﻿using System.Reflection;
+using System.Runtime.CompilerServices;
 using System.Runtime.InteropServices;
 
-// General Information about an assembly is controlled through the following
-// set of attributes. Change these attribute values to modify the information
-// associated with an assembly.
+// 有关程序集的一般信息由以下
```
Comment (martijn00) [inline OldSamples/TwitterSearch/TwitterSearch.UI.Console/Properties/AssemblyInfo.cs]: This should stay in English.

File: OldSamples/TwitterSearch/TwitterSearch.UI.Console/TwitterSearch.UI.Console.csproj [hunk 2]
```diff
@@ -1,102 +1,69 @@
...
-      <SpecificVersion>False</SpecificVersion>
-      <HintPath>..\..\Lib\Mvx\Console\Cirrious.MvvmCross.Plugins.Visibility.Console.dll</HintPath>
-    </Reference>
-    <Reference Include="Cirrious.MvvmCross.Wpf, Version=4.0.0.0, Culture=neutral, processorArchitecture=MSIL">
-      <HintPath>..\..\packages\MvvmCross.HotTuna.MvvmCrossLibraries.4.0.0-beta3\lib\net45\Cirrious.MvvmCross.Wpf.dll</HintPath>
-      <Private>True</Private>
-    </Reference>
-    <Reference Include="Newtonsoft.Json">
-      <HintPath>..\Libs\Console\Newtonsoft.Json.dll</HintPath>
+    <Reference Include="MvvmCross, Version=6.1.2.0, Culture=neutral, processorArchitecture=MSIL">
```
Comment (martijn00) [inline OldSamples/TwitterSearch/TwitterSearch.UI.Console/TwitterSearch.UI.Console.csproj]: We need to use PackageReference now.

File: OldSamples/TwitterSearch/TwitterSearch.UI.Console/packages.config [hunk 3]
```diff
@@ -1,4 +1,5 @@
 ﻿<?xml version="1.0" encoding="utf-8"?>
 <packages>
-  <package id="MvvmCross.HotTuna.MvvmCrossLibraries" version="4.0.0-beta3" targetFramework="net45" />
+  <package id="MvvmCross" version="6.1.2" targetFramework="net461" />
```
Comment (martijn00) [inline OldSamples/TwitterSearch/TwitterSearch.UI.Console/packages.config]: Please remove packages.config and use PackageReference.

File: OldSamples/TwitterSearch/TwitterSearch.UI.Wpf/MultiRegionPresenter.cs [hunk 4]
```diff
@@ -13,17 +15,16 @@ public MultiRegionPresenter(MainWindow mainWindow)
         {
             _mainWindow = mainWindow;
         }
-
-        public override void Present(FrameworkElement frameworkElement)
+        protected override void ShowContentView(FrameworkElement element, MvxContentPresentationAttribute attribute, MvxViewModelRequest request)
```
Comment (martijn00) [inline OldSamples/TwitterSearch/TwitterSearch.UI.Wpf/MultiRegionPresenter.cs]: We probably need to use the new Wpf presenter, but could leave it like this if this PR is getting too big.

File: OldSamples/TwitterSearch/TwitterSearch.UI.Wpf/TwitterSearch.UI.Wpf.csproj [hunk 5]
```diff
@@ -37,33 +38,23 @@
...
-      <HintPath>..\..\Lib\Mvx\Wpf\Cirrious.MvvmCross.Plugins.Visibility.Wpf.dll</HintPath>
-    </Reference>
-    <Reference Include="Cirrious.MvvmCross.Wpf, Version=4.0.0.0, Culture=neutral, processorArchitecture=MSIL">
-      <HintPath>..\..\packages\MvvmCross.HotTuna.MvvmCrossLibraries.4.0.0-beta3\lib\net45\Cirrious.MvvmCross.Wpf.dll</HintPath>
-      <Private>True</Private>
+    <Reference Include="MvvmCross.Platforms.Wpf, Version=6.1.2.0, Culture=neutral, processorArchitecture=MSIL">
+      <HintPath>..\packages\MvvmCross.Platforms.Wpf.6.1.2\lib\net461\MvvmCross.Platforms.Wpf.dll</HintPath>
     </Reference>
     <Reference Include="System" />
+    <Reference Include="System.Console, Version=4.0.1.0, Culture=neutral, PublicKeyToken=b03f5f7f11d50a3a, processorArchitecture=MSIL">
```
Comment (martijn00) [inline OldSamples/TwitterSearch/TwitterSearch.UI.Wpf/TwitterSearch.UI.Wpf.csproj]: Please use PackageReference

**Pull request closed without merging**